force-app/main/default/classes/SOQL_Test.cls

```diff
@@ -2519,4 +2519,26 @@ private class SOQL_Test {
         // Test
         SOQL.of(Account.SObjectType).count().preview().toInteger();
     }
+
+    @IsTest
```

Comment:
Hi @jessewheeler ! 

Your PR looks great!  🥳

Could we change the Unit Test a little bit? It's nice, but I would like to keep the same convention for all test methods. 

```java
    @IsTest
    static void groupByRelatedFormatting() {
        // Test
        String soql = SOQL.of(OpportunityLineItem.SObjectType)
            .count(OpportunityLineItem.Name, 'count')
            .groupBy('OpportunityLineItem.Opportunity.Account', Account.Id)
            .toString();

            // Verify
        Assert.areEqual(
            'SELECT COUNT(Name) count FROM OpportunityLineItem GROUP BY OpportunityLineItem.Opportunity.Account.Id',
            soql
        );
    }

    @IsTest
    static void groupByRelateToAggregated() {
        // Setup
        List<Account> accounts = insertAccounts();

        // Test
        List<AggregateResult> results = SOQL.of(Account.SObjectType)
            .count(Account.Name, 'names')
            .groupBy('Account.CreatedBy', User.Id)
            .toAggregated();

        // Verify
        Assert.areEqual(1, results.size());
    }
```

Reply:
Thanks, done!